scipy/signal/ltisys.py

```diff
@@ -273,172 +277,164 @@ def ss2zpk(A, B, C, D, input=0):
 
 
 class lti(object):
-    """Linear Time Invariant class which simplifies representation.
+    """
+    Linear Time Invariant system base class.
 
     Parameters
     ----------
-    args : arguments
+    system : lti system arguments
```

Comment:
This should just say `tuple` instead of `lti system arguments` since AFAIK it's supposed to be the data type